engine/core/src/test/java/io/knotx/fragments/engine/TaskEngineScenariosTest.java

```diff
@@ -166,18 +166,30 @@ void expectProcessingLogsInOrder(VertxTestContext testContext, Vertx vertx) thro
         fragmentEvent -> {
           assertEquals(Status.SUCCESS, fragmentEvent.getStatus());
           verifyAllLogEntries(fragmentEvent.getLogAsJson(),
-              Operation.exact("task", "first", "SUCCESS", 0),
-              Operation.range("task", "A1", "SUCCESS", 1, 4),
-              Operation.range("task", "A2", "ERROR", 1, 4),
-              Operation.range("task", "A3-fallback", "SUCCESS", 1, 4),
-              Operation.range("task", "B", "SUCCESS", 1, 4),
-              Operation.exact("task", COMPOSITE_NODE_ID, "SUCCESS", 5),
-              Operation.exact("task", "middle", "SUCCESS", 6),
-              Operation.range("task", "X", "SUCCESS", 7, 9),
-              Operation.range("task", "Y1", "SUCCESS", 7, 9),
-              Operation.range("task", "Y2", "SUCCESS", 7, 9),
-              Operation.exact("task", COMPOSITE_NODE_ID, "SUCCESS", 10),
-              Operation.exact("task", "last", "SUCCESS", 11)
+              Operation.exact("task", "first", "UNPROCESSED", 0),
```

Comment:
Thank you for your patience :) Maybe there is some better approach to manage ranges.

Reply:
One idea that comes to mind would be to do it somewhat like this:

```java
Operation.group(
   Operation.elem("task", "A", "SUCCESS"), 
   Operation.elem("task", "B", "SUCCESS"))
Operation.group(
   Operation.elem("task", "C", "SUCCESS"), 
   Operation.elem("task", "D", "SUCCESS")
```
which would mean that `A` and `B` range from 0 to 1 and `C` and `D` range from 2 to 3. I think it would be easier to make adjustments